lib/gpu_memory_service/cli/server.py

```
@@ -30,36 +31,11 @@
-
-
-def _optional_checkpoint_stop_file() -> Path | None:
-    control_dir = os.environ.get("GMS_CONTROL_DIR")
-    if not control_dir:
-        return None
-    return Path(control_dir) / "checkpoint-done"
-
-
 def main() -> None:
-    ready_file = _ready_file_path()
+    ready_file = Path(os.environ.get("GMS_SOCKET_DIR", "/tmp")) / _READY_FILE
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)